datamegh/api/storage/s3/__init__.py

```diff
@@ -0,0 +1 @@
+from .s3 import upload, download
```

Comment:
No need this. You can remove the folder `s3/` and have a file `api/storage/s3.py` it does the job.